### PR TITLE
Adding imaginary unit

### PR DESCRIPTION
**Pull request by clausmichele**
Adding a process expressing the imaginary unit. First step towards complex numbers in openEO.

### CODE REVIEW

File: proposals/i.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "id": "i",
+    "summary": "Imaginary unit",
+    "description": "The imaginary unit or unit imaginary number (i) is a solution to the quadratic equation x^2+1=0.",
+    "categories": [
+        "math > constants"
+		],
+    "parameters": [],
```
Comment (m-mohr) [inline proposals/i.json:8]: "experimental": true,

File: proposals/i.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "id": "i",
+    "summary": "Imaginary unit",
+    "description": "The imaginary unit or unit imaginary number (i) is a solution to the quadratic equation x^2+1=0.",
+    "categories": [
+        "math > constants"
+		],
```
Comment (m-mohr) [inline proposals/i.json:7]:
```suggestion
    ],
    "experimental": true,
```

File: proposals/i.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+    "id": "i",
+    "summary": "Imaginary unit",
```
Comment (m-mohr) [inline proposals/i.json:3]:
```suggestion
    "summary": "Imaginary unit (i)",
```

File: proposals/i.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    "description": "The imaginary unit or unit imaginary number (i) is a solution to the quadratic equation x^2+1=0.",
```
Comment (m-mohr) [inline proposals/i.json:4]:
```suggestion
    "description": "The imaginary unit or unit imaginary number *i* is a solution to the quadratic equation *`x^2+1=0`*.",
```

**Review by m-mohr — CHANGES_REQUESTED** [review]
Generally looks good, thanks..
Couple of comments for consistency and for linting.

Also, please make the PR against draft, not master.

**Comment by clausmichele** [comment]
Done, thanks for the suggestions Matthias.

**Review by m-mohr — APPROVED** [review]

**Review by m-mohr — CHANGES_REQUESTED** [review]
You need to recreate this PR, I think. You can't merge master into draft, otherwise other commits will be included.

Otherwise, looks good.

**Pull request closed without merging**